ui/jstests/test-main.js

```diff
@@ -1,5 +1,6 @@
+/* global SHIMS */
```

Comment:
Can you add a global directive for `REQUIRE_PATHS` too and remove it from `.jshintrc`?